### DISCUSSION TITLE
Adding plurals of unit representation strings

### DISCUSSION THREAD

**Discussion by stuchalk** [qudt/qudt-public-repo#487] [Q&A] [answered]
I am going to add unit representations for UDUNITS encoding system.  In addition to having multiple 'legal' strings for some units, e.g. tablespoons (Tbl, Tbsp, tbsp, Tblsp, tblsp), UDUNITS supports plurals of its unit strings.  Any issue with adding plurals?

**Comment by jhodgesatmb** [comment, marked as answer]
Seems unnecessary as it is only a lexical distinction and doesn’t change the representation. We should avoid this kind of change unless you plan to use skos:altLabel. Even then I wonder at the value.

Jack Hodges, Ph.D.
Arbor Studios

**Comment by steveraysteveray** [comment]
I strongly agree that there should be only a single URI for each unit (I don't think you were suggesting more, right?) and I also agree that leading people down the path of many possible labels in general just raises interoperability issues ("I couldn't find label xyz!"). Would it not be preferable for people to recognize the general practice of sticking with singular names?

**Comment by stuchalk** [comment]
This makes good sense.  Thanks.

**Comment by dr-shorthair** [comment]
`skos:altLabel` is for this. 
Or `skos:hiddenLabel` if you want to recognise it but not encourage it.
